VagrantfileA.rb

```diff
@@ -0,0 +1,6 @@
+Vagrant.configure("2") do |config|
+  config.vm.provision "file", source: "license.json", destination: "/tmp/license.json"
```

Comment:
This seems to be duplicated with the `pre_create_command: cp <%=ENV['ES_XPACK_LICENSE_FILE']%> {{vagrant_root}}` command. 

Is there a reason this can't be mounted as a volume the same way it is being done for the docker based tests?

```
      volume: 
        - <%=ENV['ES_XPACK_LICENSE_FILE']%>:/tmp/license.json
```